_layouts/media.html

```diff
@@ -182,7 +182,8 @@
 	{ id: "yO2quNzmvuY", title: "MCollective and Puppet 3 data bindings" },
 	{ id: "g5hvddwT1os", title: "Organizations and Locations implementation" },
 	{ id: "dlgv_4rr4Ws", title: "Parameterized Class support" },
-	{ id: "CmKoEHCYROQ", title: "Jenkins and Packaging" }
+	{ id: "CmKoEHCYROQ", title: "Jenkins and Packaging" },
+	{ id: "jKZppAuz6kQ", title: "PXE-less Discovery workflow" }
```

Comment:
> Can we put it at the top? I think new stuff belongs there, instead.

Oh, it was hard to tell from the source code :-)

-- 
Later,
 Lukas #lzap Zapletal